DuckDuckGo/NetworkProtectionStatusView.swift

```diff
@@ -64,38 +64,22 @@ struct NetworkProtectionStatusView: View {
     @ViewBuilder
     func inviteCodeEntry() -> some View {
         Section {
-            VStack(alignment: .leading) {
```

Comment:
Gets rid of the initial dev-designed invite entry on the dev-designed status view